diff -ru leafnode-2.0.0.alpha20090409a.luascript/system.h~ leafnode-2.0.0.alpha20090409a.luascript/system.h
--- leafnode-2.0.0.alpha20090409a.luascript/system.h~   2009-01-30 07:02:57.000000000 -0800
+++ leafnode-2.0.0.alpha20090409a.luascript/system.h    2009-11-12 21:43:03.941817635 -0800
@@ -20,9 +20,6 @@
 #include <inttypes.h>
 #elif HAVE_STDINT_H
 #include <stdint.h>
-#elif sizeof(unsigned long) == 4 && sizeof(unsigned char) == 1
-typedef unsigned long uint32_t;
-typedef unsigned char uint8_t;
 #else
 #error "I cannot figure how to define uint32_t and uint8_t."
 #endif
