--- work/kernel/nvidia-uvm/uvm_unsupported.c~	2016-10-14 09:19:09.000000000 -0700
+++ work/kernel/nvidia-uvm/uvm_unsupported.c	2016-11-14 16:57:24.658283173 -0800
@@ -171,6 +171,6 @@
 module_init(uvm_unsupported_module_init);
 module_exit(uvm_unsupported_exit);
 
-MODULE_LICENSE("MIT");
+MODULE_LICENSE("GPL");
 MODULE_INFO(supported, "external");
 
--- work/kernel/nvidia-uvm/uvm_common.c~	2016-10-14 09:19:09.000000000 -0700
+++ work/kernel/nvidia-uvm/uvm_common.c	2016-11-14 16:57:29.618249700 -0800
@@ -390,5 +390,5 @@
 MODULE_PARM_DESC(uvm_enable_builtin_tests,
                  "Enable the UVM built-in tests. (This is a security risk)");
 
-MODULE_LICENSE("MIT");
+MODULE_LICENSE("GPL");
 MODULE_INFO(supported, "external");
--- work/kernel/nvidia-drm/nvidia-drm-linux.c~	2016-10-14 09:17:27.000000000 -0700
+++ work/kernel/nvidia-drm/nvidia-drm-linux.c	2016-11-14 16:57:36.128205766 -0800
@@ -221,7 +221,7 @@
 module_exit(nv_linux_drm_exit);
 
 #if defined(MODULE_LICENSE)
-  MODULE_LICENSE("MIT");
+  MODULE_LICENSE("GPL");
 #endif
 #if defined(MODULE_INFO)
   MODULE_INFO(supported, "external");
--- work/kernel/nvidia-modeset/nvidia-modeset-linux.c~	2016-10-14 09:17:27.000000000 -0700
+++ work/kernel/nvidia-modeset/nvidia-modeset-linux.c	2016-11-14 16:57:40.714841478 -0800
@@ -1252,7 +1252,7 @@
 module_exit(nvkms_exit);
 
 #if defined(MODULE_LICENSE)
-  MODULE_LICENSE("NVIDIA");
+  MODULE_LICENSE("GPL");
 #endif
 #if defined(MODULE_INFO)
   MODULE_INFO(supported, "external");
--- work/kernel/nvidia/nv.c~	2016-10-14 09:19:10.000000000 -0700
+++ work/kernel/nvidia/nv.c	2016-11-14 16:57:46.864799974 -0800
@@ -37,7 +37,7 @@
 
 #if (NV_BUILD_MODULE_INSTANCES != 0)
 #if defined(MODULE_LICENSE)
-MODULE_LICENSE("NVIDIA");
+MODULE_LICENSE("GPL");
 #endif
 #if defined(MODULE_INFO)
 MODULE_INFO(supported, "external");
--- work/kernel/nvidia/nv-frontend.c~	2016-10-14 09:19:10.000000000 -0700
+++ work/kernel/nvidia/nv-frontend.c	2016-11-14 16:57:53.484755297 -0800
@@ -15,7 +15,7 @@
 #include "nv-frontend.h"
 
 #if defined(MODULE_LICENSE)
-MODULE_LICENSE("NVIDIA");
+MODULE_LICENSE("GPL");
 #endif
 #if defined(MODULE_INFO)
 MODULE_INFO(supported, "external");
