--- src/libavutil/x86/asm.h	2016-03-11 16:51:17.914194309 -0800
+++ src/libavutil/x86/asm.h	2016-03-11 16:48:02.000000000 -0800
@@ -72,7 +72,7 @@
 typedef int x86_reg;
 #endif
 
-#define HAVE_7REGS (ARCH_X86_64 || (HAVE_EBX_AVAILABLE && HAVE_EBP_AVAILABLE))
+#define HAVE_7REGS (ARCH_X86_64)
 #define HAVE_6REGS (ARCH_X86_64 || (HAVE_EBX_AVAILABLE || HAVE_EBP_AVAILABLE))
 
 #if ARCH_X86_64 && defined(PIC)
