--- configure~	2015-05-18 23:33:14.000000000 -0700
+++ configure	2015-05-18 23:34:15.274702779 -0700
@@ -4940,10 +4940,6 @@
 ac_compiler_gnu=$ac_cv_c_compiler_gnu
 
 
-if test "$c_compiler" = "unknown"; then
-    as_fn_error $? "Compiler GCC >= 4.7 or Clang >= 3.3 is required for C compilation" "$LINENO" 5
-fi
-
 # Check that an appropriate C++ compiler is available.
 cxx_compiler="unknown"
 ac_ext=cpp
@@ -4998,10 +4994,6 @@
 ac_compiler_gnu=$ac_cv_c_compiler_gnu
 
 
-if test "$cxx_compiler" = "unknown"; then
-    as_fn_error $? "Compiler GCC >= 4.7 or Clang >= 3.3 is required for C++ compilation" "$LINENO" 5
-fi
-
 # C/C++ Headers
 ac_ext=c
 ac_cpp='$CPP $CPPFLAGS'
